ynr/apps/api/next/filters.py

```diff
@@ -0,0 +1,10 @@
+from django_filters import filterset, filters
+
+
+class LastUpdatedFilter(filterset.FilterSet):
```

Comment:
Sorry one last comment! So this is the case where you _would_ use `Mixin` in the name - as you wouldn't use this on its own, its purpose is to be inherited elsewhere

Reply:
That makes sense, will change.